packages/vda-sbt-client/test/blockchain-api.test.ts

```diff
@@ -19,11 +33,90 @@ const createBlockchainAPI = (did: any) => {
     })
 }
 
+const claimSBT = async (
+    didAddress: string,
+    sbtType: string,
+    uniqueId: string,
+    sbtURI: string,
+    receiverAddress: string,
+    signedProof: string,
+    keyring: Keyring,
+    blockchainApi: VeridaSBTClient
+) => {
+    const signedDataMsg = ethers.utils.solidityPack(
+        ['string','address'],
+        [`${sbtType}-${uniqueId}-`, didAddress.toLowerCase()]
+    )
+    const signedData = await keyring.sign(signedDataMsg)
+
+    await blockchainApi.claimSBT(
+        sbtType,
+        uniqueId,
+        sbtURI,
+        receiverAddress.toLowerCase(),
+        signedData,
+        signedProof
+    )
+}
+
+const createTestDataIfNotExist = async (
+    trustedSigner: InterfaceDID, 
+    blockchainApi: VeridaSBTClient,
+    keyring: Keyring,
+    signedProof: string
+    ) => {
+    const orgTotalSupply = parseInt(await blockchainApi.totalSupply())
+    if (orgTotalSupply > 0) return;
```

Comment:
As you can see, if there were minted transaction before, this function'd not continue to run.